optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticType.java

```diff
@@ -23,4 +23,9 @@
      */
     public String name();
 
+    /**
+     * @return never null
+     */
+    public String formattedName();
```

Comment:
I don't really have any preference, but I'm going with C as that has the least impact for now. Fixed.